### PR TITLE
added method to get all revisions for a class

### PR DESCRIPTION
**Pull request by sniper7kills**
Created a way to get all revisions from an Object.

This is useful for displaying the last $limit User/Post/Blog/Etc changes in an admin panel.

### CODE REVIEW

**Comment by sniper7kills** [comment]
Updated to include request from #136

File: src/Venturecraft/Revisionable/Revision.php [hunk 1]
```diff
@@ -187,6 +187,22 @@ public function userResponsible()
...
+    /**
+     * Returns the object we have the history of
+     * @return Object or false
+     */
+    public function historyOf()
+    {
+        if(class_exists($class = $this->revisionable_type))
+        {
+            $dd = $class::find($this->revisionable_id);
+            dd($dd);
```
Comment (duellsy) [inline src/Venturecraft/Revisionable/Revision.php]: I'm assuming this shouldn't actually be doing a `dd` here?

**Comment by duellsy** [comment]
This all looks good to me, however can you please make the PR on the `develop` branch?

**Comment by sniper7kills** [comment]
Your right there shouldn't be. I just removed it; and I'll open a new PR on the develop branch.

**Pull request closed without merging**